manim/config/default.cfg

```diff
@@ -96,6 +96,10 @@ max_files_cached = 100
 flush_cache = False
 disable_caching = False
 
+# Default tex_template
+# --tex_template
+tex_template =
```

Comment:
**Question** : Why is it empty? Is it intended/good ? 

Cc @leotrs

Reply:
Yes, it will be interpreted as an empty string which we can then interpret as`None`. This way we use the default template automatically but the value is present in the default config.

Reply:
I see. I'm not a config expert at all, it was just weird to me. Maybe add a comment stating this ? Or can you set `None` instead ? 
Anyway, @leotrs will have the last word

Reply:
Not an expert either, as far as I know you can't set `None` explicitly. I'll see if @leotrs knows and if not I think that adding that comment is a good solution.

Reply:
You can't use `None` explicitly because .cfg files only accept strings. I think using the empty string is fine.